tests/test_geoprocessing.py

```diff
@@ -113,13 +116,11 @@ def test_reclassify_raster_missing_pixel_value(self):
             test_value: 100,
         }
         target_nodata = -1
-        with self.assertRaises(ValueError) as cm:
+        with self.assertRaises(reclass_error) as cm:
             pygeoprocessing.reclassify_raster(
                 (raster_path, 1), value_map, target_path, gdal.GDT_Float32,
                 target_nodata, values_required=True)
-        expected_message = (
-            'The following 1 raster values [-0.5] from "%s" do not have ' %
-            (raster_path,))
+        expected_message = ('The following 1 raster values [-0.5]')

```

Comment:
I think you can lose the parens here now that it's on one line.

Reply:
Parens removed in https://github.com/natcap/pygeoprocessing/pull/106/commits/8509356492a7b45beaa058e8184e09d1087603a7